it/controllers/OrchestratorControllerISpec.scala

```diff
@@ -16,49 +16,135 @@
+      }
+    }
 
-  // TODO: To be implemented when Repositroy is fully implemented
-//  "receiveSDESNotifications" should {
-//    "call FileNotificationRepositories - returns OK" in {
-//      mockFileNotificationStub(OK)
-//      val jsonToReceive: JsValue = Json.parse(
-//        """
-//          |[{
-//          |   "informationType": "type",
-//          |   "files": {
-//          |       "recipientOrSender": "recipient",
-//          |       "name": "John Doe",
-//          |       "location": "place",
-//          |       "checksum": {
-//          |           "algorithm": "beep",
-//          |           "value": "abc"
-//          |       },
-//          |       "size": 1,
-//          |       "properties": [
-//          |       {
-//          |           "name": "name",
-//          |           "value": "xyz"
-//          |       }]
-//          |   },
-//          |   "audit": {
-//          |       "correlationID": "12345"
-//          |   }
-//          |}]
-//          |""".stripMargin
-//      )
-//      val result = await(buildClientForRequestToApp(uri = "/new-notifications").post(
-//        jsonToReceive
-//      ))
-//      result.status shouldBe OK
-//    }
-//  }
+    "return error status code" when {
+      "the call to Mongo/stub has a fault" in new Setup {
+        val jsonToReceive: JsValue = Json.parse(
+          """
+            |[{
+            |   "informationType": "type",
+            |   "files": {
+            |       "recipientOrSender": "recipient",
+            |       "name": "John Doe",
+            |       "location": "place",
+            |       "checksum": {
+            |           "algorithm": "beep",
+            |           "value": "abc"
+            |       },
+            |       "size": 1,
+            |       "properties": [
+            |       {
+            |           "name": "name",
+            |           "value": "xyz"
+            |       }]
+            |   },
+            |   "audit": {
+            |       "correlationID": "12345"
+            |   }
+            |},
+            |{
+            |   "informationType": "type",
+            |   "files": {
+            |       "recipientOrSender": "recipient",
+            |       "name": "John Doe",
+            |       "location": "place",
+            |       "checksum": {
+            |           "algorithm": "beep",
+            |           "value": "abc"
+            |       },
+            |       "size": 1,
+            |       "properties": [
+            |       {
+            |           "name": "name",
+            |           "value": "xyz"
+            |       }]
+            |   },
+            |   "audit": {
+            |       "correlationID": "12345"
+            |   }
+            |}]
+            |""".stripMargin
+        )
+        val result: WSResponse = await(buildClientForRequestToApp(uri = "/new-notifications").post(
+         jsonToReceive
+        ))
+        result.status shouldBe INTERNAL_SERVER_ERROR
```

Comment:
I'm not sure how this error is caused?

Reply:
It's caused by the correlationID is a duplicate. I'll change the name of the val so it's clear what's the issue